test/bigtest/network/config.js

```diff
@@ -218,6 +218,80 @@ export default function config() {
     ];
   });
 
+  this.post('erm/sas', (_, request) => {
+    const body = JSON.parse(request.requestBody);
+    return this.create('agreement', body).attrs;
+  });
+
+  this.get('/erm/org', () => []);
+
+  this.get('/erm/contacts', {
+    results: []
+  });
+
+  this.get('/users', (schema, request) => {
+    console.log(request.queryParams, 'nattuga');
+    return schema.contacts.all();
+  });
+
+  this.get('note-links/domain/agreements/type/agreement/id/:id', () => { });
+
+  this.get('/note-types', () => { });
+
+  this.get('/erm/sas/:id', (schema, request) => {
+    return schema.agreements.find(request.params.id);
+  });
+
+  this.get('/erm/sas/:id/resources/current', (schema, request) => {
```

Comment:
Double-check all of these merge conflicts you had. You've just duplicated the endpoint handlers for these. I cleaned them up and fixed them to work with multiple matching resources previously.